lib/src/header_parser/sub_parsers/compounddecl_parser.dart

```diff
@@ -271,7 +271,20 @@ int _compoundMembersVisitor(clang_types.CXCursor cursor,
         if (mt.baseType is UnimplementedType) {
           parsed.unimplementedMemberType = true;
         }
-
+        if (mt is PointerType && mt.child is NativeFunc) {
```

Comment:
It's possible that there can be multiple levels of typedef nesting (for mt or mt.child), You can use Type.baseType and/or Type.typedefType to get the underlying type)